lib/gcov/README.md

```diff
@@ -0,0 +1,77 @@
+# Unikraft Library for Profiling and Test Coverage
+
+This library introduces initial setup with config options for console, binary file, and memory output. It takes advantage of gcc-13's newest feature of Profiling and Test Coverage in Freestanding Environments.
```

Comment:
Here and elsewhere: `s/gcc-13/GCC-13/g`. Also although it's newest now, in the future it will look stale. Prefer something like `GCC's Profiling and Test Coverage...`. This takes care of the next comment about minimum version.